### ISSUE TITLE
eliminate dependencies and remove addon: AddThis addon for Plone 1.3.3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#495]

**Comment by paregorios** [comment]
This addon is still `active` in `prefs_install_products_form` on 5x staging. @sauzher can it be deactivated and removed as part of the migration from 4.3 to 5.1, or is that a manual step between 5.1 and 5.2?

**Comment by paregorios** [comment]
- [ ] uninstall on staging
- [ ] if ok uninstall on production
- [ ] adjust 5.1 buildout to not include

**Comment by chiruzzimarco** [comment]
@alecpm I merged my PR on the plone5.1 branch, pulled the changes on the migration server and ran the script:

```
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_account_name')
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_activated')
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_button_visible')
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_chicklets')
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_data_track_addressbar')
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_data_track_clickback')
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_load_asynchronously')
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_script_url')
('Deleted registry key:', 'collective.addthis.interfaces.IAddThisSettings.addthis_url')
('Deleted registry key:', 'collective.addthis.socialmediasources')
('Deleting control panel action:', 'addthis-settings', '-', u'Configure AddThis')
Removed collective.addthis browser layer.
```

Then i've rerun the buildout with `bin/buildout -c migration.cfg`, restarted the services with `supervisorctl restart migration_zeoclient1 migration_zeoserver` as `root` and removed the `collective.addthis` egg manually.

@paregorios Now there should be no trace of the `collective.addthis` in the 5.1 build.

**Comment by paregorios** [comment]
@chiruzzimarco I see that this is now no longer visible on the isaw migration server. It is still active on isaw production. Will this be a problem for the migration process when the current production database is brought forward for final migration? I ask because [we originally said that we should remove this addon from the 4.3 series staging and production as well](https://github.com/isawnyu/isaw.web/issues/495#issuecomment-2755285850).

**Comment by sauzher** [comment]
> I don't see any issues in the migration process related to this addon ([@sauzher](https://github.com/sauzher) can you confirm?), we may consider running the same procedure on the 4.3 servers indeed.

Yes, it's removal can be done before migration, just like others cleanings.
